README.md

```diff
@@ -431,7 +431,11 @@ A number of options are available to allow you to customise the SDK:
 
   - `showCountrySelection` (boolean - default: `false`)
 
-    If only one document type is preselected that is not a passport document, the issuing country selection screen can still be displayed by setting this to `true`.
+    The `showCountrySelection` option controls what happens when **only a single document** is preselected in `documentTypes` It has no effect when the SDK has been set up with multiple documents preselected.
+
+    The country selection screen is never displayed for a passport document.
+
+    By default, for non-passport document types, if only one document type is preselected, the issuing country selection screen is also not displayed. If you would like to have this screen displayed still, set `showCountrySelection` to `true`.
```

Comment:
IMO it would make it slightly clearer to reword it as follow. Feel free to challenge it though.
```
By default, if only one document type is preselected, and the document type is not `passport`, the country selection screen will not be displayed.

Reply:
@seewah thoughts on this? I personally don't feel strongly for either one.

Reply:
actually i was going to make a similar comment. not a big deal, so if you can change that would be good. If you think we need to close this asap, thats also fine!

Reply:
OK, will make the change. Will need to resolve a merge conflict with the CHANGELOG anyway. 👍